controller/src/main/java/io/pravega/controller/server/rest/generated/api/ApiException.java

```diff
@@ -1,6 +1,6 @@
 package io.pravega.controller.server.rest.generated.api;
 
-@SuppressWarnings("all")
+
```

Comment:
please dont change generated code

Reply:
This was not changed intentionally, but on regeneration these values did not get added... adding them back now.

Reply:
fixed